src/components/inputs/JuiceSwitch.tsx

```diff
@@ -20,9 +21,14 @@ export const JuiceSwitch = ({
         onChange={onChange}
       />
       {label && (
-        <label htmlFor={switchId} className="text-black dark:text-grey-200">
-          {label}
-        </label>
+        <div>
+          <label htmlFor={switchId} className="text-black dark:text-grey-200">
+            {label}
+          </label>
+          {description ? (
```

Comment:
Support a description in JuiceSwitch, note both description and non-description instances look fine:

<img width="701" alt="Screen Shot 2023-07-16 at 12 39 43 pm" src="https://github.com/jbx-protocol/juice-interface/assets/96150256/0cfddcfb-c6c4-44e3-8e14-ca42da76837b">